varsim_somatic.py

```diff
@@ -189,7 +192,8 @@
                       "--mean_fragment_size", str(args.mean_fragment_size),
                       "--sd_fragment_size", str(args.sd_fragment_size),
                       "--disable_rand_vcf",
-                      "--disable_rand_dgv"] + other_varsim_opts + vcf_arg_list + filter_arg_list + disable_sim_arg_list \
+                      "--disable_rand_dgv",
+		      "--sv_insert_seq", args.sv_insert_seq.name] + other_varsim_opts + vcf_arg_list + filter_arg_list + disable_sim_arg_list \
```

Comment:
This should only be added to the command-line if it is specified on the command-line for varsim_somatic.